_episodes/l1-02-committing-history.md

```diff
@@ -136,6 +136,7 @@ Initialized empty Git repository in /home/username/recipe/.git/
 The path you see in the output will vary depending on your operating system.
 
 > ## `master` and `main` branches
+> A branch is a specific version of the state and history of the work in the repo.
 > Traditionally, the default branch name whenever you `init` a repository was `master`.
 > However, the sensitivity of the online community has shifted lately and some tools,
 > like GitHub, use now `main` as the default name instead. You can read the rationale
```

Comment:
I know this isn't something you've changed, but I'm wondering if maybe we should rework this a little bit. I feel like we should just write it working under the assumption that `main` is the default branch name, and title the section "The `main` branch" or something. It could be a bit confusing if we mix up explaining the concept of a main branch with the history of how it used to be referred to as `master` etc. Afterwards in a separate paragraph, we should mention in passing that traditionally the main branch was called `master` and give the link to the explanation as to why GitHub et al. made the change, because it is important that they are aware that many repos use `master` as their main branch. I'm not a fan of the word "sensitivity" here, just in case somebody misreads it as implying that people are being overly sensitive about this issue.

Reply:
On your first point - I assumed that `git` versions >  2.28 would automatically create the `main` branch upon `init`, but this doesn't seem to be the case: `master` is created, and some text appears stating the `master` vs `main` issue. So I think it's worth keeping this here.

I agree with your second - I don't think it's that the community has become more sensitive, but more aware.

Reply:
Oops, my bad. I didn't read this in its proper context, but it is actually a separate thing they need to be aware of. Sorry!

Re the "sensitivity" thing, I'd just just do straight reportage: say that GitHub and various other orgs have changed to using `main` and give the link to the explanation for those who are interested in why.

Reply:
I think it is important to explain the master/main issue here. About the second part, please do rephrase as you see fit!